jxon.js

```diff
@@ -253,7 +308,12 @@
           if (isNodeJs) {
             oParentEl.setAttribute(sName.slice(1), vValue);
           }
-        // do nothing: special handling of xml namespaces is done via createElementNS()
+          // do nothing: special handling of xml namespaces is done via createElementNS()
+        } else if (sName === opts.attrPrefix + 'xmlProlog') {
```

Comment:
I think this should really exist outside the `loadObjTree` function. Otherwise, one cannot have an actual property named "xmlProlog", as in `<foo xmlProlog="bar" />`.